apps/smeargle/src/components/Filter/FilterDetailed/FilterComboBox.tsx

```diff
@@ -0,0 +1,125 @@
+import { For, Show, createEffect } from 'solid-js';
+import { Transition } from 'solid-transition-group';
+import clsx from 'clsx';
+import { FilterComboBoxProps, FilterName } from '../types';
+import { filterPlaceholder } from '../data';
+import { useComboBoxInput } from './useComboBoxInput';
+import { useFilterContext } from '../FilterProvider';
+import { TeaserRequestParams } from 'tonemato-types';
+import SearchIcon from '~/components/Icons/SearchIcon';
+import { useTeasers } from '~/hooks/useTeasers';
+
+export default function FilterComboBox(
+  props: FilterComboBoxProps<TeaserRequestParams[FilterName]>
+) {
+  const {
+    handleInputChange,
+    inputValue,
+    handleInputKeyDown,
+    selectOption,
+    isInputInOptions,
+    handleBlur,
+  } = useComboBoxInput(props);
+
+  const [filterState, { setActive }] = useFilterContext();
+
+  const [, { isLoading }] = useTeasers();
```

Comment:
Sometimes we have [_, otherVar] and sometimes [, otherVar]. Also in useFilterDetailed.ts but i can't add a suggestion in this, lel.
```suggestion
  const [_, { isLoading }] = useTeasers();
```